### DISCUSSION TITLE
There is a delay in calling the delay method itself

### DISCUSSION THREAD

**Discussion by six1203** [celery/celery#8424] [Q&A]
Most of the calls to the delay method are 1ms, and the method call ends，Sometimes the delay method is called, and it will take 5s to return

Isn't calling delay simply triggering the execution of an asynchronous task?

**Comment by six1203** [comment]
This is my celery startup command
 
    celery -A worker worker -P threads -c 8 -l info
    celery -A worker beat -l debug
 
The server is 8 core 8g，one k8s replicas

The message queue uses redis

**Comment by six1203** [comment]
I want to know what happened to call the delay method? how can i check

**Comment by auvipy** [comment]
always try to start with a conversation please
